src/main/java/com/exasol/bucketfs/BucketFactory.java

```diff
@@ -11,5 +11,5 @@ public interface BucketFactory {
      * @param bucketName  name of the bucket
      * @return bucket
      */
-    public Bucket getBucket(final String serviceName, final String bucketName);
+    Bucket getBucket(final String serviceName, final String bucketName);
```

Comment:
Please don't change this. Our team convention is to explicitly use `public` in interfaces.